src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -59,6 +67,62 @@ internal void InitializeLicense(string licenseText)
             license = foundLicense;
         }
 
+        static bool FirstTimeUser()
+        {
+            // Check for the presence of HKCU:Software/NServiceBus
```

Comment:
The formatting of this comment is inconsistent the one below. I suggest changing this to:

`//Check for the presence of HKCU\SOFTWARE\NServiceBus`